### PR TITLE
Education Information page

### PR DESCRIPTION
**Pull request by vishwasanavatti**

### CODE REVIEW

**Review by dalex135 — APPROVED** [review]
Reviewed but it is unable to check the icons as I have a problem with Icons. 
Thanks for the alignment modifications of icons.

File: frontend/src/app/forms/steps/education-info-entry/education-info-entry.component.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+      <div>
+        <ion-icon item-left name="time-outline" size="large"></ion-icon>
+        <ion-item style="display: inline-block;">
+          <ion-label position="floating" color="medium"
+            >Graduation Date</ion-label
+          ><ion-datetime
+            displayFormat="YYYY"
+            max="2030"
+            min="2000"
```
Comment (MBuchalik) [inline frontend/src/app/forms/steps/education-info-entry/education-info-entry.component.html]: Maybe, the minimum is a bit too strict :) How about people who have graduated 25 years ago?

File: frontend/src/app/forms/steps/education-info/education-info.component.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      },
+      swipeToClose: true,
+      presentingElement: await this.modalController.getTop(),
+    });
+    /**
+     * save and cancel of education info form
+     */
+    modal.onDidDismiss().then((val) => {
+      if (val.data && val.data.canSubmitData) {
+        educationInfoAdd.markAsPristine();
```
Comment (MBuchalik) [inline frontend/src/app/forms/steps/education-info/education-info.component.ts]: Why is `markAsPristine` needed here?

File: frontend/src/app/home/home.page.html [hunk 3]
```diff
@@ -7,18 +7,15 @@
...
         ></path>
       </svg>
-
       <h1>Welcome to Consileon</h1>
       <h3>We make the difference</h3>
     </div>
-
     <div class="button-container">
       <button class="fancy-button" (click)="startApplication()">
         Apply now
```
Comment (MBuchalik) [inline frontend/src/app/home/home.page.html]: I don't see any reason why you have modified this file... (Also, I believe having some blank lines between the items makes it better to read.)

**Comment by MBuchalik** [comment]
I just had a really brief look at this PR. (So this is not an in-depth review.) In general, it looked really good to me! 🎉  I added only a few really minor comments.

File: frontend/src/app/forms/steps/education-info/education-info.component.ts (hunk 2 above)
Comment (Bharathravi1995) [reply frontend/src/app/forms/steps/education-info/education-info.component.ts]: hey @MBuchalik  that has to be removed. It was checked in accidentally while trying out to check whether a user modified any thing in UI. But we found another way to do this.

File: frontend/src/app/forms/steps/education-info-entry/education-info-entry.component.html (hunk 1 above)
Comment (vishwasanavatti) [reply frontend/src/app/forms/steps/education-info-entry/education-info-entry.component.html]: Not sure if people graduated 25 years ago will be attending job fair. Anyways will make min="1980" which could be enough.

File: frontend/src/app/home/home.page.html (hunk 3 above)
Comment (vishwasanavatti) [reply frontend/src/app/home/home.page.html]: Am not sure why is this file modified. I had checked this file for reference. will revert changes.

**Review by Nasser-Eddin-Nasser — COMMENTED** [review]
I ran the code locally and checked the functions of the App, it works perfectly :)

**Comment by MBuchalik** [comment]
Alright 👍 Would be cool if you could merge it today, so that I have time to update the feature documentation. :)

**Pull request merged**